lib/transfer.js

```diff
@@ -181,26 +183,41 @@ function _shouldTransferFile(file) {
   return false;
 }
 
+
 /**
  * Request file from URL and upload with filesAdapter
  * @param  {Ibject}   file     the file info object
  */
-function _transferFile(file) {
-  return new Promise(function(resolve, reject) {
+function _transferFile(file) 
+{
+  return new Promise(function(resolve, reject) 
+  {
     if (config.transferTo == 'print') {
       console.log(file.url);
       // Use process.nextTick to avoid max call stack error
       return process.nextTick(resolve);
     }
-    request({ url: file.url, encoding: null }, function(error, response, body) {
-      if (_requestErrorHandler(error, response)) {
-        return reject(error);
-      }
-      config.filesAdapter.createFile(
-        file.newFileName, body, response.headers['content-type']
-      ).then(function() {
-        return _changeDBFileField(file);
-      }).then(resolve, reject);
-    });
+
+	// exists?
+	var localFileName = config.filesAdapter._getLocalFilePath(file.newFileName) ;
+	if ( !fs.existsSync(localFileName) )
```

Comment:
this'll only work if you're using the local file system.  in my case I am copying all assets to s3 by using the https://github.com/parse-server-modules/parse-server-s3-adapter